FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/ApiRepository.kt

```diff
@@ -141,12 +141,12 @@ interface ApiRepository : BaseApiRepository {
    * Grab a reply token before uploading a password protected message
    *
    * @param context Interface to global information about an application environment.
-   * @param domain A company domain.
-   * @param domain OIDC token.
+   * @param baseFesUrlPath A base FES URL path.
+   * @param idToken OIDC token.
    */
   suspend fun getReplyTokenForPasswordProtectedMsg(
     context: Context,
-    domain: String,
+    baseFesUrlPath: String,
```

Comment:
Here it's called `baseFesUrlPath` but in `getClientConfiguration` it's called `customFesUrl` - both methods should be getting the same string, and both should be called the same thing. consistency

Reply:
Agree. It can have the same structure. I will change it.